src/plugin/cesium/sync/featureconverter.js

```diff
@@ -1892,3 +1938,16 @@ plugin.cesium.sync.FeatureConverter.prototype.setEyeOffset = function(primOffset
 plugin.cesium.sync.FeatureConverter.prototype.getEyeOffset = function() {
   return this.eyeOffset_;
 };
+
+
+/**
+ * Convert a style's line dash to 16 bit int
+ * @param {!ol.style.Style} style
+ * @return {number}
+ */
+plugin.cesium.sync.FeatureConverter.prototype.getDashPattern = function(style) {
+  var stroke = style.getStroke();
+  var dashPattern = stroke != null ? stroke.getLineDash() : undefined;
+  var id = /** @type {os.style.styleLineDashOption} */ (os.style.dashPatternToOptions(dashPattern)).id;
+  return plugin.cesium.LINE_STYLE_OPTIONS[id].csPattern;
```

Comment:
This seems like a somewhat risky return statement. Is `plugin.cesium.LINE_STYLE_OPTIONS[id]` always guaranteed to be defined?